api/tests/unit/domain/read-models/JuryCertificationSummary_test.js

```diff
@@ -31,4 +32,89 @@ describe('Unit | Domain | Models | JuryCertificationSummary', () => {
       });
     });
   });
+
+  describe('#isActionRequired', () => {
+    context('when at least one issue report requires action', () => {
+      it('should return true', () => {
+        // given
+        const juryCertificationSummary = new JuryCertificationSummary({ certificationIssueReports: [
+          domainBuilder.buildCertificationIssueReport({ category: CertificationIssueReportCategories.FRAUD }),
+        ] });
+
+        // when
+        const isRequired = juryCertificationSummary.isActionRequired();
+
+        // then
+        expect(isRequired).to.be.true;
+      });
+
+      it('should return false', () => {
+        // given
+        const juryCertificationSummary = new JuryCertificationSummary({ certificationIssueReports: [
+          domainBuilder.buildCertificationIssueReport({
+            category: CertificationIssueReportCategories.LATE_OR_LEAVING,
+            subcategory: CertificationIssueReportSubcategories.SIGNATURE_ISSUE,
+          }),
+        ] });
+
+        // when
+        const isRequired = juryCertificationSummary.isActionRequired();
+
+        // then
+        expect(isRequired).to.be.false;
+      });
+    });
+  });
+
+  describe('#hasScoringError', () => {
+    context('when assessment result has a scoring error', () => {
+      it('should return true', () => {
+        // given
+        const juryCertificationSummary = new JuryCertificationSummary({ status: AssessmentResult.status.ERROR });
+
+        // when
+        const hasScoringError = juryCertificationSummary.hasScoringError();
+
+        // then
+        expect(hasScoringError).to.be.true;
+      });
+
+      it('should return false', () => {
```

Comment:
**question (non bloquante)** 
Idem ici, le contexte `when assessment result has a scoring error` me paraît bizarre avec `should return false` ...